package/MDAnalysis/coordinates/H5MD.py

```diff
@@ -727,3 +728,93 @@ def has_forces(self):
     @has_forces.setter
     def has_forces(self, value: bool):
         self._has['force'] = value
+
+    def __getstate__(self):
+        state = self.__dict__.copy()
+        del state['_particle_group']
+        return state
+
+    def __setstate__(self, state):
+        self.__dict__ = state
+        self._particle_group = self._file['particles'][
+                                list(self._file['particles'])[0]]
+
```

Comment:
```suggestion
        self[self.ts.frame]

```
This is not needed in this specific case, but just to fit to be broader API.